bgpd/bgp_vty.c

```diff
@@ -10921,9 +10921,51 @@ static struct cmd_node bgp_evpn_node =
 
 static void community_list_vty (void);
 
+static void
+bgp_ac_neighbor (vector comps, struct cmd_token *token)
+{
+  struct bgp *bgp;
+  struct peer *peer;
+  struct listnode *lnbgp, *lnpeer;
+
+  bool ipv4 = !strcmp(token->text, "A.B.C.D");
```

Comment:
I'm actually not sure why I'm not using `token->type` there... changing it to that.